pkg/certificate/providers/certmanager/types.go

```diff
@@ -57,6 +55,9 @@ type CertManager struct {
 
 	// crLister is used to list CertificateRequests in the given namespace.
 	crLister cmlisters.CertificateRequestNamespaceLister
+
+	// dynamic config retrieval
```

Comment:
nit: It does not seem like the comment adds any value and should not be needed since this is not exported.

Reply:
Going to remove this in a follow-up.